data/src/Cardano/Data/StrictFingerTree.hs

```diff
@@ -0,0 +1,278 @@
+{-# LANGUAGE BangPatterns               #-}
```

Comment:
This is not the most recent version of `StrictFingerTree`, you're missing https://github.com/input-output-hk/cardano-prelude/pull/146. Please copy the latest version.

Reply:
I now have the most recent version of strict sequence and strict fingertree.